apps/agreement/README.md

```diff
@@ -0,0 +1,5 @@
+# Agreement
```

Comment:
I think it would also be nice to start introducing a more standard format for these readmes with information on their status, like the [voting-connectors do](https://github.com/aragonone/voting-connectors/blob/master/apps/voting-aggregator/readme.md) (for now, we can just say it's in heavy development and pending audit).